tools/tf2openapi/cmd/main.go

```diff
@@ -1,42 +1,61 @@
 package main
 
 import (
-	"flag"
 	"github.com/golang/protobuf/proto"
-	"github.com/kubeflow/kfserving/tools/tf2openapi/generator"
 	pb "github.com/kubeflow/kfserving/tools/tf2openapi/generated/protobuf"
+	"github.com/kubeflow/kfserving/tools/tf2openapi/generator"
+	"github.com/spf13/cobra"
```

Comment:
Do we already depend on this explicitly? If we don't let's take an explicit dependency.

Reply:
added to Gopkg.toml & Gopkg.lock